extension/src/sidebar.css

```diff
@@ -104,25 +132,157 @@ a.promnesia-visited {
 #visits .datetime {
     white-space: nowrap;
     float: right;
-
-    color: #666;
-
     cursor: pointer;
+    font-family: monospace;
 }
 
-
 /* TODO not great, but quickest way to render it neatly */
 /* TODO not sure which order users expect first? */
 #visits .time {
-    /* color: blue; */
-    margin-right: 1em;
+    padding-right: 0.5em;
+    margin-right: 0.5em;
+    color: var(--time-color);
 }
+
 #visits .date {
-    /* color: red; */
     width: 11ch; /* TODO not sure how to alighn them properly across all of boxes... */
     display: inline-block; /* otherwise width doesn't work on span */
     text-align: end;
+    border-radius: 6px;
+    padding: 2px 4px;
+    color: var(--color2);
+    background: var(--date-bg);
+}
 
+#promnesia-sidebar {
+    display: flex;
+    flex-flow: column;
+    height: 100%;
+    margin: 0;
+    background: var(--bg);
+    overscroll-behavior: contain;
 }
 
+#promnesia-sidebar-header {
+}
 
+#promnesia-sidebar-toolbar {
+    padding: 6px 8px 0 8px;
+    border-bottom: 1px solid var(--bg);
+    overflow: hidden;
+    background: var(--header-bg);
+}
+
+#promnesia-sidebar-toolbar .button.active {
+    border-color: var(--source-color);
+}
+
+#promnesia-sidebar-toolbar .button-close {
+    float: right;
+    border-radius: 999px;
+    padding: 4px;
+}
+
+#promnesia-sidebar-container {
+    flex-grow: 1;
+    overflow-y: auto;
+}
+
+#promnesia-sidebar-filters {
+    text-transform: capitalize;
```

Comment:
it capitalize word `filter: ` (BTW, why not "Filter:" in initial js?) + source buttons....
can you put screen?

Reply:
ah -- I think it didn't work for some reason, but seem that you removed it now anyway
Rither way, imo if the user wanted to capitalize source names they would just name them from the capital letter. Also it makes it a bit misleading for debugging etc since the source names in data would be different from what they appear as.
renaming to `Filter: ` is fine though, I just have a very relaxed attitude towards capital letters xD